tests/Unit/ExecutorServiceTemplateTest.php

```
@@ -0,0 +1,41 @@
+    $compose = file_get_contents($templatePath);
+
+    expect($compose)
+        ->toContain('ghcr.io/rhyssullivan/executor-selfhost:${EXECUTOR_VERSION:-latest}')
+        ->toContain('SERVICE_URL_EXECUTOR_4788')
+        ->toContain('EXECUTOR_WEB_BASE_URL=${SERVICE_URL_EXECUTOR_4788}')
+        ->toContain('executor-data:/data')
+        ->toContain('http://127.0.0.1:4788/api/health');
+
+    foreach (['service-templates.json', 'service-templates-latest.json'] as $templateFile) {
+        $templates = json_decode(
+            file_get_contents(__DIR__."/../../templates/{$templateFile}"),
```

PHPStan (2.2.8)

[warning] Parameter `#1` $json of function json_decode expects string, string|false given.


(argument.type)